onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -378,6 +388,9 @@ def _where_input_wrangler(
     "nn.functional.conv1d": core_ops.aten_conv1d,
     "nn.functional.conv2d": core_ops.aten_conv2d,
     "nn.functional.conv3d": core_ops.aten_conv3d,
+    "nn.functional.nll_loss": (nn_ops.aten_nll_loss, _nll_loss_input_wrangler),
+    # "nn.functional.nll_loss2d": (nn_ops.aten_nll_loss2d, _nll_loss_input_wrangler),  # not in OPS_DB
```

Comment:
Could you please add them into extra_opinfo.py?

Reply:
remove nll_loss2d